MC/bin/o2dpg_sim_workflow.py

```diff
@@ -807,6 +822,46 @@ def createRestDigiTask(name, det='ALLSMALLER'):
   TOFMatchQCtask['cmd'] = 'o2-qc --config json://${O2DPG_ROOT}/MC/config/QC/json/tofMatchedTracks_ITSTPCTOF_TPCTOF.json --remote-batch TOFMatchQC.root ' + getDPL_global_options()
   workflow['stages'].append(TOFMatchQCtask)
 
+if includeAnalysis:
+   # Configuration
+   analysisdir = "Analysis"
+   analysislabel = "Analysis"
+   if not isdir(analysisdir):
+      mkdir(analysisdir)
+
+   def addAnalysisTask(tag, cmd, output, needs=[AOD_merge_task['name']]):
+      """
+      Function to add O2Physics analysis task to the workflow and upload the results on the CCDB
+      """
+      if type(output) == str:
+         output = [output]
+      AnalysisTasks = createTask(name=f"Analysis_{tag}",
+                                      needs=needs,
+                                      cwd=analysisdir,
+                                      lab=[analysislabel], cpu=1, mem='2000')
+      renameOutput = ""
+      AnalysisTaskOutput = []
+      for i in output:
+         i = i.strip(".root")
+         renameOutput += f" && mv {i}.root {i}_{tag}.root "
+         AnalysisTaskOutput.append(f"{i}_{tag}.root")
+      AnalysisTasks['cmd'] = f"{cmd} {renameOutput}"
+      workflow['stages'].append(AnalysisTasks)
+
+      # Uploading results to ccdb
+      if QUALITYCONTROL_ROOT is None:
+         return
+      for i in AnalysisTaskOutput:
+         AnalysisFinalizetask = createTask(name=f"Analysis_finalize_{tag}_{i}",
+                                           needs=[AnalysisTasks['name']],
+                                           cwd=analysisdir, lab=[analysislabel+"Upload"], cpu=1, mem='2000')
+         AnalysisFinalizetask['cmd'] = f"o2-qc-upload-root-objects --input-file ./{i} --qcdb-url ccdb-test.cern.ch:8080 --task-name Analysis{tag} --detector-code TST --provenance qc_mc --pass-name passMC --period-name SimChallenge --run-number 49999"
```

Comment:
```suggestion
         AnalysisFinalizetask['cmd'] = f"o2-qc-upload-root-objects --input-file ./{i} --qcdb-url ccdb-test.cern.ch:8080 --task-name Analysis{tag} --detector-code AOD --provenance qc_mc --pass-name passMC --period-name SimChallenge --run-number 49999"
```
Perhaps you could put AOD as the detector, unless you really treat it as a test (TST) for now.